lldb/source/Plugins/SymbolFile/DWARF/DWARFFormValue.cpp

```diff
@@ -578,6 +583,7 @@ bool DWARFFormValue::IsBlockForm(const dw_form_t form) {
   case DW_FORM_block1:
   case DW_FORM_block2:
   case DW_FORM_block4:
+  case DW_FORM_data16:
```

Comment:
Do we want `IsBlockForm` to return true for a `DW_FORM_data16`?

Reply:
We might simply because this is represented the same way. Many systems don't support `uint128_t` types, so we can't just add one of those, so we need to use a byte pointer. Or we can add a `bool DWARFFormValue::IsLargeConstant(const dw_form_t form)` if we don't want to re-use the block stuff.

Reply:
I think that the block form will be the most convenient way to access this just because not all systems have uint128_t.